test/JDBC/input/BABEL_4389-vu-prepare.sql

```diff
@@ -0,0 +1,61 @@
+-- This will force creation of orphan entries in babelfish_view_def table if possible
+CREATE VIEW BABEL4389V_1 as SELECT 1
+GO
+DROP VIEW BABEL4389V_1
+GO
+
+CREATE VIEW BABEL4389V1 as SELECT 1
+GO
+DROP VIEW BABEL4389V1
+GO
+
+CREATE SCHEMA babel_4389_schema1
+GO
+
+CREATE VIEW babel_4389_schema1.BABEL4389V_1 as SELECT 1
+GO
+DROP VIEW babel_4389_schema1.BABEL4389V_1
+GO
```

Comment:
In verify script, we are trying to create view in different schema which does not prove if we are able to create the view with same name after dropping it. So update this test to create view in default schema or update verify script to match this same.

Reply:
Applicable for other views also